content/docs/bdd/_index.md

```diff
@@ -6,4 +6,107 @@ menu: main
 iconClass: fas fa-comments
 ---
 
-Behaviour-Driven Development is a set of collaborative practices for agile teams.
+Behaviour-Driven Development (BDD) is the software development process that Cucumber was built to support.
+
+There's much more to BDD than just using Cucumber.
+
+# What is BDD?
+
+BDD is a way for software teams to work that aims to close the gap between business people and technical people by:
+
+* Encouraging collaboration across roles to build shared understanding of the next part of the problem to be solved
+* Working in rapid, small iterations to increase feedback and flow of value
+* Producing system documentation that is automatically validated against the system's behaviour
+
+We do this by focussing collaborative work around concrete, real-world, examples that illustrate how we want the system to behave. We use those examples to guide us from concept through to implementation.
+
+## BDD and agile
+
+We assume that your team are using some kind of agile methodology already, planning work in small increments of value like [User Stories]. BDD does not replace your existing agile process, it enhances it.
```

Comment:
I like the links to terminology.

Reply:
:)

Yeah I'm hoping that can grow and become really useful.